README.md

```diff
@@ -27,7 +27,32 @@ Download the binary for your platform from the [releases](https://github.com/ran
 wget https://github.com/rancher/terraform-provider-rke/releases/download/x.x.x/terraform-provider-rke_<platform>
 ```
 
-Rename the binary to to remove the platform ending (so it looks like `terraform-provider-rke`)and move it to your terraform plugin directory (`~/terraform.d/plugins/linux_amd64`) or a custom directory. If using a custom directory use the `-plug-in=/my/custom/path` when running `terraform init` to specify the path.
+1. rename it to `terraform-provider-rke_v0.14.1`
+1. put it to `~/.terraform.d/plugins`
+1. remove the `.terraform` in ur `project_directory`
```

Comment:
Not needed to remove `.terraform` folder, in fact it's a installation option

Reply:
But, If I do not remove this folder, `terraform init` will not be forced to download the plugin

Reply:
What do you mean?? Until the plugin is officially supported by Hashicorp, it has to be downloaded manually.

Reply:
I mean that if the plugin inside `.terraform` does not correct, for instance, the current https://github.com/rancher/quickstart/pull/62 is still open and the quick start example is not yet started. U not to remove the folder `.terraform` in order to trigger force update.

Reply:
Copying provider binary to `.terraform/plugins/<OS>_<ARCH>/terraform-provider-rke_<VERSION>` is working fine for me.

Reply:
Hey @rawmind0 
I mean if the common provider is broken. Just like the PR https://github.com/rancher/quickstart/pull/62. The only way to force it to update the official provider plugin is remove the plugin folder within the project directory. I am not talking about the third-party plugin folder directory.

Reply:
@AlphaWong, `.terraform` folder can be used. I guess would be better to let the user choose to use  `~/.terraform.d/plugins` or `.terraform`.

Reply:
This mostly solves issue #199. I think it's worth adding an explanation of the naming conventions expected here. For example, I was unsure whether I could use one of the RCs since they did not follow the `X.Y.Z` format exactly--would Terraform recognize them as greater than `0.14.1`, or at all? 

Ultimately, a bit more experimentation would have answered that question, but my guess is that many potential Rancher users would have already been scared off by that point. Remember that K8s is super-intimidating for anyone just arriving to it because it changes so fast.

 Speaking from my own experience, I pretty much ditch any path that involves any resistance, because there's almost always some other way that might be newer, easier, closer to my use case. I might have ditched Rancher if I hadn't found dealing with kops' hard dependency on Route53 to be even more annoying than figuring out what was going on with this issue.

Reply:
@borenstein, we are working to onboard this module on hashicorp, promote it as official provider and avoid manual installation steps. Unfortunately, the process is being longer due current situation, and we are waiting for hashicorp answer. 

> In this case, even after I figured out what was happening, I was unsure whether I could use one of the RCs since they did not follow the X.Y.Z format exactly--would Terraform recognize them as greater than 0.14.1, or at all?

Terraform uses this library to deal with versioning, https://github.com/hashicorp/go-version taking into account version suffix as `-rc5`.  Anyway, today we've cut new v1.0.0 release. 

Thanks for not ditching Rancher. :)